kube/src/config/mod.rs

```diff
@@ -40,6 +40,8 @@ pub struct Config {
     pub(crate) identity: Option<(Vec<u8>, String)>,
     /// Stores information to tell the cluster who you are.
     pub(crate) auth_info: AuthInfo,
+    /// Proxy URL. Not used unless `proxy` and `proxy-native-tls`/`proxy-rustls-tls` features are enabled.
+    pub proxy_url: Option<String>,
```

Comment:
I think this is the correct thing to do as well. Export it to at least let it be parseable, but have the doc comment.